src/dash-adapter.js

```diff
@@ -1149,6 +1149,28 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     return false;
   }
 
+  /**
+   * Gets the segment duration of the stream
+   * @return {number} - Segment duration in seconds
+   */
+  get liveDuration() {
+    return this._getLiveEdge();
```

Comment:
dont you need to add the segment duration like in hls and native?

Reply:
No, they are probably included it already.